etc/gcerc

```diff
@@ -85,6 +85,12 @@ alias gcescp="scp -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no"
 # conda.
 alias use.conda3="namemunge PATH ~/opt/conda3/bin"
 alias use.conda2="namemunge PATH ~/opt/conda2/bin"
+# aliases are not expanded when the shell is not interactive.  This cause error
+# message "use.conda3: command not found" dumped to stdout, and may result in
+# the other issues. For example, when non-interactive ssh connection is
+# established to scale up parallel computing, the shell may access the message
+# from the stdout.  Use shopt to always enable aliases expanded.
+shopt -s expand_aliases
 if [ -d ~/opt/conda3/bin ]; then use.conda3; fi
```

Comment:
Maybe a more dependable resolution is to replace ``use.conda3`` with ``namemunge PATH ~/opt/conda3/bin`` here?  The replaced line would look more consistent around ``then``:

```bash
if [ -d ~/opt/conda3/bin ]; then namemunge PATH ~/opt/conda3/bin; fi
```

And it will be good if gcerc limits additional settings done to users' environment.  Forcing ``shopt -s expand_aliases`` may be too aggressive to users, especially when they really want to turn off aliases for any unknown reason.